docs/pages/includes/discovery/reference/aws-iam/elasticache.mdx

```diff
@@ -0,0 +1,28 @@
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "ElastiCacheDiscovery",
+            "Effect": "Allow",
+            "Action": "elasticache:DescribeReplicationGroups",
+            "Resource": "*"
+        },
+        {
+            "Sid": "ElastiCacheFetchMetadata",
+            "Effect": "Allow",
+            "Action": [
+                "elasticache:DescribeCacheClusters",
+                "elasticache:DescribeCacheSubnetGroups",
+                "elasticache:ListTagsForResource"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+| Statement | Purpose |
+| --------- | ------- |
+| `ElastiCacheDiscovery` (required) | Discover ElastiCache replication groups. |
+| `ElastiCacheFetchMetadata` (optional) | Import AWS tags and additional metadata for each database as Teleport database labels. |
```

Comment:
I am not sure how I feel about marking these optional. Without importing tags, you must have a wildcard matcher. I recall helping a customer debug an issue where RDS proxies were not discovered and turned out to be missing permissions to fetch tags. Even not for tag filtering, I do not see why we wouldn't import more labels to provide better context. 

For simplicity, I would group all of these in one as required. For the API that describes extra endpoints, maybe we can have a paragraph (similar to the one in RDS section) that the API is only required when you want to discover those specific endpoints. 

Or maybe the word "optional" is too weak. 

Let me know what you think.

Reply:
I'm glad you pointed that out - I assumed that required means "it can't possibly work without these permissions", whereas optional it technically could work without those permissions, but you are right that if the db service relies on imported tags in its matcher then "optional" is misleading.
I will just remove the required/optional phrasing.

I still want to keep the separate statements though, because I want to describe the different purposes of each.